tests/config/database.php

```diff
@@ -1,35 +1,22 @@
 <?php
 
-$mongoHost = env('MONGO_HOST', 'mongodb');
-$mongoPort = env('MONGO_PORT') ? (int) env('MONGO_PORT') : 27017;
-$mysqlPort = env('MYSQL_PORT') ? (int) env('MYSQL_PORT') : 3306;
-
 return [
-
     'connections' => [
-
         'mongodb' => [
             'name' => 'mongodb',
             'driver' => 'mongodb',
-            'host' => $mongoHost,
+            'dsn' => env('MONGODB_URI', 'mongodb://127.0.0.1/'),
             'database' => env('MONGO_DATABASE', 'unittest'),
```

Comment:
Note my earlier suggestion to rename this to `MONGODB_DATABASE` for consistency.

Reply:
Good point, changed.